collector/system.go

```diff
@@ -81,46 +81,49 @@ func (c *SystemCollector) Collect(ch chan<- prometheus.Metric) error {
 type Win32_PerfRawData_PerfOS_System struct {
 	ContextSwitchesPersec     uint32
 	ExceptionDispatchesPersec uint32
+	Frequency_Object          uint64
 	ProcessorQueueLength      uint32
 	SystemCallsPersec         uint32
 	SystemUpTime              uint64
 	Threads                   uint32
+	Timestamp_Object          uint64
 }
 
 func (c *SystemCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, error) {
-	var dst []Win32_PerfRawData_PerfOS_System
-	if err := wmi.Query(wmi.CreateQuery(&dst, ""), &dst); err != nil {
+	var raw []Win32_PerfRawData_PerfOS_System
+	if err := wmi.Query(wmi.CreateQuery(&raw, ""), &raw); err != nil {
```

Comment:
Minor nitpick: Now that you aren't pulling in multiple counters, might as well revert this name to `dst`, since that is what we use everywhere else (in truth, I'm not sure why we use that name, though? 😄  )